nbs/15_gsa_gls/30-real_phenotypes/05-group_gwas_by_n_variants.ipynb

```diff
@@ -0,0 +1,1599 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    assert df_stats["min"] &gt; 0.99
</pre><p>why do df_stats need to be larger than 0.99?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/phenoplier/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>

Reply:
These `assert`s are my visual assertions. In this case, I want to make sure that if I run the notebook again, those pairs of traits share at least 0.99 of the variants.